geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/data/RedisSortedSetCommandsFunctionExecutor.java

```diff
@@ -43,4 +43,10 @@ public long zadd(RedisKey key, List<byte[]> scoresAndMembersToAdd, ZAddOptions o
   public byte[] zscore(RedisKey key, byte[] member) {
     return stripedExecute(key, () -> getRedisSortedSet(key, true).zscore(member));
   }
+
+  @Override
+  public byte[] zincrby(RedisKey key, byte[] increment, byte[] member) {
+    return stripedExecute(key,
+        () -> getRedisSortedSet(key, false).zincrby(getRegion(), key, increment, member));
+  }
```

Comment:
Could we add a test to `AbstractHitsMissesIntegrationTest` verifying that stats should be updated for `ZINCRBY`?

Reply:
lol I already forgot about it since we talked about it 10 minutes ago 😅 wilco

Reply:
done